cmd/jaeger/internal/extension/remotesampling/extension.go

```diff
@@ -140,7 +140,7 @@ func (ext *rsExtension) Shutdown(ctx context.Context) error {
 
 	if ext.httpServer != nil {
 		if err := ext.httpServer.Shutdown(ctx); err != nil {
```

Comment:
There is no need for these if statements if we're not wrapping errors, just add directly to the array (including nil)

Reply:
Doing so reduces the need for extra unit tests